monkey/infection_monkey/exploit/mssqlexec.py

```diff
@@ -23,6 +24,7 @@ class MSSQLExploiter(HostExploiter):
     _EXPLOITED_SERVICE = "MSSQL"
     _TARGET_OS_TYPE = ["windows"]
     LOGIN_TIMEOUT = 15
```

Comment:
Consider using LONG_REQUEST_TIMEOUT from `monkey/common/common_consts/timeouts.py:3` Regardless of protocol, uploading monkey and executing commands should take a similar time.

Reply:
So if monkey get's smaller we could decrease upload timeout everywhere at once, for example